main.tf

```diff
@@ -8,7 +8,7 @@ locals {
  */
 resource "aws_cloudwatch_log_group" "cw_b2_fsbackup" {
   name              = local.app_name_and_env
-  retention_in_days = 45
+  retention_in_days = 60
```

Comment:
Is this the new recommendation? Just wondering what I should use the next time I go there.

Reply:
Not necessarily.  You said someone wanted 45 days.  45 isn't valid but 60 is so it meets (and exceeds) the "requirement" for 45 days.